### ISSUE TITLE
代码运行问题？？

### ISSUE THREAD

**Issue by 591569024** [mocherson/ImageGCN#1]
model.py文件中from layers import *因为没有layers的代码而报错，您能否提供代码？万分感谢！！

**Comment by mocherson** [comment]
谢谢提醒，layers.py 已更新。

**Comment by 591569024** [comment]
**Preprocessing.** Each image in the dataset is resized to 256 × 256, and then cropped to 224 × 224 at the center  for fast processing. We normalized the image by mean ([0:485; 0:456; 0:406]) and standard deviation ([0:229; 0:224; 0:225]) of the images from ImageNet.
我阅读了您的代码，但是没有找到对数据预处理部分的代码，为了完整的复现您的结果，如果方便的话您能提供这部分的代码吗？

**Comment by mocherson** [comment]
可以按如下代码进行预处理，
from torchvision import transforms
import pandas as pd

transform=transforms.Compose([transforms.Resize((256,256)), transforms.CenterCrop((224,224))])

df = pd.read_csv('/home/hddraid/shared_data/chest_xray8/Data_Entry_2017.csv')

for name in df['Image Index']:
    image = Image.open(join('/home/hddraid/shared_data/chest_xray8/images/images/', name))
    transform(image).save('/home/hddraid/shared_data/chest_xray8/images/cropedimages/'+name)

**Comment by 591569024** [comment]
**4.6. Disease Localization**
您能提供一下Disease Localization部分的代码吗？得到这部分结果的代码我并没有找到，谢谢！

**Comment by mocherson** [comment]
Disease Localization已经添加bbox.py  .

**Comment by qqto2008** [comment]
您好能提供下您使用的数据集，感谢

**Comment by mocherson** [comment]
数据集ChestX-ray14可以从这里下载
https://nihcc.app.box.com/v/ChestXray-NIHCC

**Comment by qqto2008** [comment]
你好有个HGCN的方法报错，您是引用了什么库么
<img width="494" alt="Screen Shot 2020-04-29 at 10 43 15 pm" src="https://user-images.githubusercontent.com/16584453/80597201-db61af00-8a6a-11ea-9e6b-0e361595ca52.png">

**Comment by mocherson** [comment]
class MedGAE(nn.Module): 这个类和本项目无关，删了即可。我这个文件是为了与其他项目兼容所以实现了多个model的类。

**Comment by qqto2008** [comment]
好的 谢谢

**Comment by qqto2008** [comment]
您好 我想问一下 这么大的数据 您是用什么机器配置跑的，花了多长时间，我自己的电脑完全跑不起来，还有个bug是如果epoch用default是10的话跑到第4个会有个floating point error，但是如果把epoch 改2的话就不会有这个error。 万分感谢

**Comment by mocherson** [comment]
我用的K40 GPU 跑的，大概跑两三天，不会报错。你完整的报错信息是什么，floating point error 感觉应该是和机器的浮点数精度有关。

**Comment by qqto2008** [comment]
<img width="544" alt="Screen Shot 2020-05-01 at 1 55 00 pm" src="https://user-images.githubusercontent.com/16584453/80780747-5b932c00-8bb3-11ea-91f3-9f9ba4432d4d.png">
完整的信息就是这个

**Comment by qqto2008** [comment]
我用的是mac，用的是cpu跑的，我觉得可能是这个问题。万分感谢

**Comment by qqto2008** [comment]
你好 我我想问下 文件里面有个BBox_List, 想问一下 这个文件是用来干什么的，谢谢
